### ISSUE TITLE
Successful login message on register

### ISSUE THREAD

**Issue by Twonox** [Xephi/AuthMeReloaded#1052]
[url=http://www.noelshack.com/2017-01-1483531302-2017-01-04-130119.png][img]http://image.noelshack.com/minis/2017/01/1483531302-2017-01-04-130119.png[/img][/url]
Is it normal that it have successful login message on register ?

**Comment by sgdc3** [comment]
I think it's fine @ljacqu

**Comment by ljacqu** [comment]
@sgdc3 Agreed as this is a border case. Handling this case specifically = ugly code that will have to be maintained. 

However, @Twonox @sgdc3, I have a counter proposition—as an enhancement (and an _easy pick story_ :wink:) we could distinguish between when a user is _force_ logged in vs. logged in because of his own actions.
- If the player logs in because he used /login, everything stays as before (i.e. if successful -> "Successful login!" message)
- If the player is logged in due to other circumstances (after registration, API use, /authme forcelogin), he receives another message (e.g. "You have been logged in")

-> Same could be done for unregistering. If an admin unregisters a player, the player will get the message "Successfully unregistered!" which is a little weird.

**Comment by Twonox** [comment]
Mhmm why not.

**Comment by sgdc3** [comment]
It's fine for me ;)

**Comment by Twonox** [comment]
@ljacqu I don't appreciate that
<a href="http://www.noelshack.com/2017-01-1483531302-2017-01-04-130119.png"><img src="http://image.noelshack.com/minis/2017/01/1483531302-2017-01-04-130119.png" border="0" alt="1483531302-2017-01-04-130119.png - envoi d'image avec NoelShack" title="1483531302-2017-01-04-130119.png"/></a>

and that
<a href="http://www.noelshack.com/2017-01-1483653602-2017-01-05-225420.png"><img src="http://image.noelshack.com/minis/2017/01/1483653602-2017-01-05-225420.png" border="0" alt="1483653602-2017-01-05-225420.png - envoi d'image avec NoelShack" title="1483653602-2017-01-05-225420.png"/></a>

The better thing for me it's to sent only one message: only successful register message and session message, without successful login message.
I will appreciate if you undo this (and if you find a solution to maintain easily the code).

**Comment by ljacqu** [comment]
> I will appreciate if you **undo** this
 
Do you mean to say that this came when we added the registration type settings? I will check, but the fix will probably only make it in 5.3.

**Comment by Maxetto** [comment]
I guess this is related, Console login messages are printed more than once.
At register:
```
29.01 16:24:39 [Server] INFO ProGirl100 registered (IP)
29.01 16:24:39 [Server] INFO ProGirl100 logged in!
```

At server switch:
```
29.01 16:47:41 [Server] INFO GalaxyDoritoMC logged in!
29.01 16:47:42 [Server] INFO GalaxyDoritoMC logged in!
```

Login messages printed twice (or even three times) are due to FastLogin and BungeeAuthMeBridge sending new login thru the API.

**Comment by SaitDev** [comment]
Also got the message `Logged-in due to Sesstion Reconnection`, but just got it few times not everytime I loggin.
Using version 5.2 latest release. sessions.enabled is false. No bungge server and no FastLogin.
Did it cause by other plugins that call authme api?

**Comment by ljacqu** [comment]
@SaitDev this kind of seems like the only possibility; if this problem continues please don't hesitate to create a separate issue about this.
